client/my-sites/themes/current-theme/index.tsx

```diff
@@ -107,7 +107,7 @@ class CurrentTheme extends Component< CurrentThemeProps > {
 														'components-button',
 														'current-theme__' + this.props.name
 													) }
-													primary={ option.label.toLowerCase() === 'customize' }
+													primary={ name === 'customize' }
```

Comment:
I wonder why this wasn't just `name` previously? 🤔

Reply:
Maybe `name` is localized?

Reply:
No, I think name is simply the object key from https://github.com/Automattic/wp-calypso/blob/6bff72177cde497405e1018a4722bdde862b0294/client/my-sites/themes/theme-options.js#L190

As best I can tell, `customize` is the only theme action used here as a primary action, so checking the name instead of the label should be okay.